packages/driver/types/internal-types.d.ts

```diff
@@ -58,6 +58,10 @@ declare namespace Cypress {
     (action: 'clear:cookies', fn: () => void)
     (action: 'cross:origin:cookies', fn: (cookies: SerializableAutomationCookie[]) => void)
     (action: 'before:stability:release', fn: () => void)
+    (action: '_log:added', fn: (attributes: ObjectLike, log: Cypress.Log) => void): Cypress
+    (action: 'log:added', fn: (attributes: ObjectLike, log: Cypress.Log) => void): Cypress
+    (action: '_log:changed', fn: (attributes: ObjectLike, log: Cypress.Log) => void): Cypress
+    (action: 'log:changed', fn: (attributes: ObjectLike, log: Cypress.Log) => void): Cypress
```

Comment:
`log:changed` and `log:added` are already in the public types.

```suggestion
    (action: '_log:added', fn: (attributes: ObjectLike, log: Cypress.Log) => void): Cypress
    (action: '_log:changed', fn: (attributes: ObjectLike, log: Cypress.Log) => void): Cypress
```

Reply:
[`ccbeea0` (#28531)](https://github.com/cypress-io/cypress/pull/28531/commits/ccbeea0bfaf24861ebb0d38c53ee6cba94bbddd2)